functions/snapshot/fetch-users.ts

```diff
@@ -1,7 +1,7 @@
 import fetch, { type Response } from "node-fetch";
 
 export const usersUrl =
-    "https://webhook.site/74a21588-7d3b-4eb7-91ed-11a1e498af1c";
+    "https://webhook.site/6b495c5d-95b3-40b1-b85d-49dcf76a3d44";
```

Comment:
Out of interest, did we need to create a new API endpoint at webhook because the old one expired, or something like that?

Reply:
Yes, the old one had expired and the webhook site provided me with a new one. The contents of which I changed to match the original.